website/src/client/configs/sdk.tsx

```diff
@@ -4,12 +4,12 @@
  * This is useful when deploying the website  with preliminary support for a new SDK version.
  */
 export const versions: { [version: string]: boolean } = {
-  '38.0.0': true,
+  '38.0.0': false, // dropped with sdk 42.0.0
```

Comment:
I think this will fail as the types from website and snack-sdk are now different. Should just remove this as SDK 38 won't work anymore after this PR

Reply:
Removed it and used the `SDKVersion` type in the object!